bindgen/visitor/ScalaFrontendAction.h

```diff
@@ -12,14 +12,15 @@
  */
 class ScalaFrontendAction : public clang::ASTFrontendAction {
   public:
-    explicit ScalaFrontendAction(IR &ir);
+    ScalaFrontendAction(IR &ir, LocationManager &locationManager);
```

Comment:
`LocationManager` is declared in `ir/location` so it makes sense to add it as a member to IR to avoid passing it where `IR` is passed, etc. Better yet might be to have `Location` under `ir/` but have the `LocationManager` be part of the `TreeVisitor` so it has access to the `SourceManager`. We don't need the manager once we are done with the AST because we have the locations themselves.